packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-stack-builder/auth-cognito-stack-builder.ts

```diff
@@ -762,7 +774,7 @@ export class AmplifyAuthCognitoStack extends cdk.Stack implements AmplifyAuthCog
   /**
    * Creates Custom lambda resource to update 3rd party providers on userpool
    */
-  createHostedUIProviderCustomResource(): void {
+  deleteExistingHostedUIProviderCustomResource(): void {
```

Comment:
nit: comment says "Creates" but function name is "delete"

Reply:
darn, let me change the comment